src/util/is-valid-url-test.ts

```diff
@@ -0,0 +1,14 @@
+import { isValidUrl } from "utils/is-valid-url";
```

Comment:
`src/util/is-valid-url-test.ts` needs to be `src/util/is-valid-url.test.ts` with a `.test` to be picked up by jest

Reply:
already uopdated